src/main/controllers/DescribeWhatHappenedController.ts

```diff
@@ -45,13 +46,14 @@ export default class DescribeWhatHappenedController {
     },
   };
 
-  constructor() {
+  constructor(private logger: LoggerInstance) {
     this.form = new Form(<FormFields>this.describeWhatHappenedFormContent.fields);
   }
 
-  public post = (req: AppRequest, res: Response): void => {
+  public post = async (req: AppRequest, res: Response): Promise<void> => {
     setUserCase(req, this.form);
     handleSessionErrors(req, res, this.form, PageUrls.TELL_US_WHAT_YOU_WANT);
+    await handleUploadDocument(req, req.body.claimSummaryFile, this.logger);
```

Comment:
Could you please set the response from the call to the claimSummaryFile property on the userCase